hoomd/data/typeconverter.py

```diff
@@ -150,35 +150,42 @@ def __str__(self):
         return "OnlyIf({})".format(str(self.cond))
 
 
-class OnlyType(_HelpValidate):
+class OnlyTypes(_HelpValidate):
     """Only alllow values that are instances of type.
 
     Developers should consider the `collections.abc` module in using this type.
-    In general `OnlyType(Sequence)` is more readible than the similar
+    In general `OnlyTypes(Sequence)` is more readible than the similar
     `OnlyIf(lambda x: hasattr(x, '__iter__'))`.
     """
-    def __init__(self, type_, strict=False,
+    def __init__(self, types, strict=False,
```

Comment:
Actually this can't be that bad since you already renamed the class everywhere that it's being used as part of this PR... so maybe this is worth doing? I still leave it up to you, but I do think this would make for a cleaner interface.

Reply:
I think this is fine, but I would do it in is own PR. This is already a bit of track of the original point. I do like that idea though.